From: Jeroen Dekkers <jeroen@dekkers.ch>
Date: Mon, 11 Mar 2013 19:37:45 +0100
Subject: Link libSOGo with libdl

---
 SoObjects/SOGo/GNUmakefile.preamble | 4 ++++
 1 file changed, 4 insertions(+)

Index: sogo/SoObjects/SOGo/GNUmakefile.preamble
===================================================================
--- sogo.orig/SoObjects/SOGo/GNUmakefile.preamble	2014-02-10 12:44:32.917427822 +0100
+++ sogo/SoObjects/SOGo/GNUmakefile.preamble	2014-02-10 12:44:54.000000000 +0100
@@ -49,6 +49,10 @@
 SOGo_LIBRARIES_DEPEND_UPON += -lcrypt
 endif
 
+ifeq ($(findstring gnu, $(GNUSTEP_HOST_OS)), gnu)
+SOGo_LIBRARIES_DEPEND_UPON += -ldl
+endif
+
 ADDITIONAL_TOOL_LIBS += \
         -L$(GNUSTEP_OBJ_DIR)/ \
         -lSOGo \
